tests/service-manager.js

```diff
@@ -7,6 +7,7 @@ const processes = [];
 let stoppingAll = false;
 
 const stopAll = (code) => {
+  console.log(`[e2e] Sending kill signals to all servers due to code ${code}`);
```

Comment:
I moved this, because otherwise it's really not clear that stopAll is being fired